drivers/wifi/nrf700x/zephyr/src/zephyr_wpa_supp_if.c

```diff
@@ -22,6 +22,8 @@ LOG_MODULE_DECLARE(wifi_nrf, CONFIG_WIFI_LOG_LEVEL);
 
 /* TODO: Move this to driver_zephyr.c */
 extern struct wpa_supplicant *wpa_s_0;
+struct wpa_signal_info *signal_info;
```

Comment:
This should be in the `vif_ctx_zep` to support multiple VIFs.

Reply:
Done.